src/content/docs/hi/install/auto.mdx

```diff
@@ -138,3 +138,23 @@ Astro आपकी `src/` निर्देशिका में लाइव
 📚 **अपनी साइट परिनियोजित करें:** हमारी [परिनियोजन मार्गदर्शिका](/hi/guides/deploy/) में जानें कि वेब पर Astro परियोजना कैसे बनाएं और परिनियोजित करें।
 
 📚 **अपने कोडबेस को समझें:** हमारे [परियोजना संरचना मार्गदर्शिका](/hi/core-concepts/project-structure/) में Astro के परियोजना की संरचना के बारे में और जानें।
+
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 136 to 141]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->```suggestion
```
I think these can just be deleted now, since they've been replaced with the cards?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/withastro/docs/6423?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->